.github/workflows/ci.yml

```diff
@@ -26,10 +26,19 @@ jobs:
     name: Tests
     timeout-minutes: 5
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # how to say "not these" so we don't miss anything?
+        # waiting on a an api from vitest for querying
+        # the list of tests ahead of time before running them.
+        slow-test:
```

Comment:
this is kinda silly atm, but it cuts the test time down proportionally per split.